conans/client/conan_api.py

```diff
@@ -531,16 +517,22 @@ def info_build_order(self, reference, settings=None, options=None, env=None,
                          install_folder=None):
         reference, profile = self._info_get_profile(reference, install_folder, profile_name, settings,
                                                     options, env)
-        graph = self._manager.info_build_order(reference, profile, build_order, remote, check_updates)
+
+        recorder = ActionRecorder()
```

Comment:
It seems that methods not using json output/recorder, shouldn't need to instantiate it?